contracts/orderbook/IOrderBook.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.8.0;
+
+interface IOrderBook {
+    enum OrderType {
+        LIMIT_OPEN,
+        LIMIT_CLOSE,
+        MARKET_STOP
+    }
+    struct Order {
+        bytes32 loanID;
+        bytes32 orderID;
+        uint256 amountReceived;
+        uint256 leverage;
+        uint256 loanTokenAmount;
+        uint256 collateralTokenAmount;
+        uint64 timeTillExpiration;
```

Comment:
I see you using uint64 which is not alligned. also a couple of bool and bytes. see if you can memory align them

Reply:
for example leverage surely doesn't have to be uint256 but this might complicate things in other parts. so see if you can optimize

Reply:
will do